TodoApp/TodoApp.Api/Models/Item.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace TodoApp.Api.Models
+{
+    public class Item
+    {
+        public string Id { get; set; }
```

Comment:
C# have native type for guid, use it instead of string as you are using guid in your FE app.